src/main/java/io/confluent/connect/jdbc/sink/DbStructure.java

```diff
@@ -112,7 +117,7 @@ void create(
       final FieldsMetadata fieldsMetadata
   ) throws SQLException {
     if (!config.autoCreate) {
-      throw new ConnectException(
+      throw new SchemaMismatchException(
```

Comment:
I think, that since for now, we will be doing the same thing with each exception (if we create another one), it might make more sense to make it a broader term and instead add a new exception if later we want more granularity.

Reply:
Sounds good to me 👍